scripts/directory_setup.py

```diff
@@ -39,12 +39,16 @@ def makedir(path):
   'labor': {'formula': 'damages ~ -1 + anomaly + np.power(anomaly, 2)'},
   'AMEL_m1': {'formula': 'damages ~ -1 + anomaly + np.power(anomaly, 2)'},
   'CAMEL_m1_c0.20': {'formula': 'damages ~ -1 + anomaly + np.power(anomaly, 2) + gmsl + np.power(gmsl, 2)'}},
-  'save_path': str(output)}
+  'save_path': str(output),
+  'gas_conversions':{
```

Comment:
I see the gas conversions in both this script and the create_config.py, is that necessary? something for another PR